### PR TITLE
*: bump to 3.0.0-beta.0

### PR DESCRIPTION
**Pull request by gyuho**
Tested locally

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ import (
 var (
 	// MinClusterVersion is the min cluster version this etcd binary is compatible with.
 	MinClusterVersion = "2.2.0"
```
Comment (xiang90) [inline version/version.go]: minCluserVersion = 2.3.0

**Comment by gyuho** [comment]
@xiang90 Just fixed. PTAL.

**Comment by xiang90** [comment]
LGTM!

**Comment by gyuho** [comment]
Thanks. Merging in greens.

**Pull request merged**